ibridges/path.py

```diff
@@ -90,14 +90,26 @@ def absolute(self) -> IrodsPath:
         """
         # absolute path
         if len(self._path.parts) == 0:
-            return IrodsPath(self.session, self.session.home)
-        if self._path.parts[0] == "~" or self._path.parts[0] == ".":
+            return IrodsPath(self.session, self.session.cwd)
+        if self._path.parts[0] == "~":
```

Comment:
Ah, I see, when a cwd is set, IrodsPath(session, "some_file.txt") will become cwd/some_file.txt.
this needs some good documentation.
I verufied, that "~" will always point to the home and "." always refers to the cwd.